core/src/models/Import.ts

```diff
@@ -33,6 +33,8 @@ export interface ImportProfileProperties {
   [key: string]: any;
 }
 
+const IMPORT_CREATORS = ["schedule", "run", "api"] as const;
```

Comment:
Can schedules create imports directly? I believe they're always created through runs 🤔

Also, I know we have an action that creates imports with `creatorType` of `api`- mainly out of curiosity, what was the use case for that? I don't think I see it being called anywhere.

Reply:
That's a very good point - Keeping that API around makes even less sense for our Data Engineer persona.  Removing it in https://github.com/grouparoo/grouparoo/pull/2209 (a PR into this PR)